src/redux/HomeContext/HomeProvider.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useReducer } from "react";
+import HomeContext, { HomeState, defaultHomeState } from "./HomeContext";
+import { ActionType, Action } from "./ActionType";
+
+type Props = {
+  children: React.ReactNode;
+};
+
+const homeReducer = (state: HomeState, action: Action): HomeState => {
+  switch (action.type) {
```

Comment:
Store in Home.tsx as local variables using useState. Creating a context is redundant

Reply:
Fixed.